common/djangoapps/course_modes/views.py

```diff
@@ -258,15 +259,18 @@ def get(self, request, course_id, error=None):  # lint-amnesty, pylint: disable=
         # Route to correct Track Selection page.
         # REV-2133 TODO Value Prop: remove waffle flag after testing is completed
         # and happy path version is ready to be rolled out to all users.
+        # REV-2415 TODO Value Prop: remove temporary logs that contain [Track Selection Check]
         if VALUE_PROP_TRACK_SELECTION_FLAG.is_enabled():
             if not error:  # TODO: Remove by executing REV-2355
                 if not enterprise_customer_for_request(request):  # TODO: Remove by executing REV-2342
+                    LOG.info('[Track Selection Check] Rendering VP template')
```

Comment:
That's a good point (the amount of logs per day), I added this just out of curiosity to how much we are still hitting the legacy template. I will remove it because it's not necessary for my ticket.